### ISSUE TITLE
iPad 2 concentration label artifacts

### ISSUE THREAD

**Issue by phet-steele** [phetsims/neuron#122]
@jbphet I've so far only seen this on two iPad 2s (Witten & Tycho). With concentrations enabled, zooming in and out leaves small artifacts below the brackets of the intracellular Na+ concentration:

![20160225_100537](https://cloud.githubusercontent.com/assets/5863899/13327556/981f85f8-dba7-11e5-9c70-9d2e0af2b41a.jpg)

@ariel-phet JB has said that this would take about an hour to fix, up to you if this is pressing enough to block publication.
For phetsims/tasks/issues/522

**Comment by phet-steele** [comment]
Troubleshooting information:
Name: ‪Neuron‬
URL: http://www.colorado.edu/physics/phet/dev/html/neuron/1.0.0-rc.5/neuron_en.html
Version: 1.0.0-rc.5 2016-02-22 20:22:46 UTC
Features missing: fullscreen
User Agent: Mozilla/5.0 (iPad; CPU OS 9_2_1 like Mac OS X) AppleWebKit/601.1.46 (KHTML, like Gecko) Version/9.0 Mobile/13D15 Safari/601.1
Language: en-us
Window: 1024x671
Pixel Ratio: 1/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 IMGSGX543-124.1)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.00)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 8 uniform: 128
Texture: size: 4096 imageUnits: 8 (vertex: 8, combined: 8)
Max viewport: 4096x4096
OES_texture_float: true
Dependencies JSON: {"assert":{"sha":"d4909677","branch":"HEAD"},"axon":{"sha":"6f3c2e01","branch":"HEAD"},"babel":{"sha":"586da1ac","branch":"master"},"brand":{"sha":"c9282935","branch":"HEAD"},"chipper":{"sha":"1d1d5d32","branch":"HEAD"},"dot":{"sha":"7a28b1a9","branch":"HEAD"},"griddle":{"sha":"3adb78bd","branch":"HEAD"},"joist":{"sha":"a8fdef22","branch":"HEAD"},"kite":{"sha":"4a947119","branch":"HEAD"},"neuron":{"sha":"3f072389","branch":"HEAD"},"phet-core":{"sha":"73a034c6","branch":"HEAD"},"phetcommon":{"sha":"d712ace0","branch":"HEAD"},"scenery":{"sha":"47143cdd","branch":"HEAD"},"scenery-phet":{"sha":"a7f975d1","branch":"HEAD"},"sherpa":{"sha":"be8c4fc9","branch":"HEAD"},"sun":{"sha":"64555735","branch":"HEAD"},"tandem":{"sha":"07fbe58c","branch":"HEAD"}}

**Comment by ariel-phet** [comment]
@jbphet yes, I think we should fix...we have put enough effort in that we should just finish it off to the best of our knowledge.

**Comment by jbphet** [comment]
I've added an invisible rectangle behind each of the readouts in order to fix this issue.  As an experiment, I also overrode the bounds calculation (```computeShapeBounds```) of the each of the readouts to match these dilated rectangles, and this improved performance on iPad 2 + iOS 9.2.1 (PhET device name = "Witten") by 1/2 to 1 FPS.  I ended up keeping this experiment due to the improved performance.
